app/modules/ads/index.js

```diff
@@ -179,20 +142,14 @@ class Ads {
   finishFirstCall() {
     const { btfBlockerService } = window.Wikia.adEngine;
 
-    if (this.isAdStackEnabled()) {
-      btfBlockerService.finishFirstCall();
-      fanTakeoverResolver.resolve();
-    }
+    btfBlockerService.finishFirstCall();
```

Comment:
What if there is no ad stack on page?

Reply:
Shouldn't we mark it as private? (if it's executed in safe place)

Reply:
Marked.